autoware_launch/config/localization/yabloc/expressway/graph_segment.param.yaml

```diff
@@ -0,0 +1,18 @@
+/**:
+  ros__parameters:
+      # graph_node selects a road surface area from around this height
+      target_height_ratio: 0.85
+
+      #
+      target_candidate_box_width: 15
+
+      # graph_segment_node will pickup additional roadlike areas
```

Comment:
Please fix these spell-check errors
![image](https://github.com/autowarefoundation/autoware_launch/assets/44218668/76f17e6d-7196-4bc1-b2d3-619152c95d36)

(I've added ximgproc here: https://github.com/tier4/autoware-spell-check-dict/pull/542, but for others I think its better to just change them to something else)